bridge/contracts/DutchAuction.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.11;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "contracts/utils/ImmutableAuth.sol";
+import "contracts/interfaces/IValidatorPool.sol";
+
+/// @custom:salt DutchAuction
+/// @custom:deploy-type deployUpgradeable
+contract DutchAuction is Initializable, ImmutableFactory, ImmutableValidatorPool {
+    uint256 private immutable _startPrice = 1000000 * 10**18;
+    uint256 private immutable _ethdkgValidatorCost = 1200000 * 2 * 100 * 10**9; // Exit and enter at an estimated price of 100 gwei
+    uint8 private immutable _decay = 16;
+    uint16 private immutable _scaleParameter = 100;
```

Comment:
Is there a reason why these are `immutable` rather than `constant`? Also - would the pre-calculated values be cheaper to use than putting the calculations on the first 2?
If these values are passed in the constructor then this contract could be used in other use cases down the line

Reply:
Changed immutables to constants since we are not initializing them from constructor. On the other hand testing with pre-calculated values did not result in any gas cost reduction. Regarding initializing immutables from constructor instead of upgrading the contract to change the constants...What do you think about it @vtleonardo?